tools/java/common/pom.xml

```diff
@@ -49,6 +49,12 @@
     </plugins>
   </build>
 
+  <repositories>
+    <repository>
+      <id>protobuf-java-lib</id>
+      <url>file://${project.basedir}/../../../java/lib</url>
+    </repository>
+  </repositories>
   <dependencies>
     <dependency>
       <groupId>junit</groupId>
```

Comment:
There are some path issues with the junit that's checked in. We should fix that.